tests/acceptance/v1.controller.acceptance.ts

```diff
@@ -1982,5 +1982,40 @@ describe('V1 controller (acceptance)', () => {
       expect(response.body).to.have.property('error')
       expect(response.body.error.message).to.startWith('Restricted endpoint: contract address not allowed')
     })
+
+    // eslint-disable-next-line mocha/no-exclusive-tests
+    it.only('invokes POST /v1/{appId} and successfully relays a request only through the altruist', async () => {
+      const pocket = pocketMock.object()
+      const logSpy = sinon.spy(logger, 'log')
+
+      ;({ app, client } = await setupApplication(pocket, { ALTRUIST_ONLY_CHAINS: '0041' }))
+
+      const response = await client
+        .post('/v1/sd9fj31d714kgos42e68f9gh')
+        .send({ method: 'eth_blockNumber', id: 1, jsonrpc: '2.0' })
+        .set('Accept', 'application/json')
+        .set('host', 'eth-mainnet-x')
+        .expect(200)
+
+      expect(response.headers).to.containDeep({ 'content-type': 'application/json' })
+      expect(response.body).to.have.properties('id', 'jsonrpc', 'result')
+      expect(parseInt(response.body.result, 16)).to.be.aboveOrEqual(0)
```

Comment:
ignorant question here. Ive seen this in like all our tests. But what exactly is it telling us? Feels pretty flimsy.

Reply:
is an example response from a node, that is the general format but the values inside doesn't really matter as long as they are the ones to be expected